test/fixtures/class-generics-and-stereotypes/parse-out.default

```diff
@@ -0,0 +1,687 @@
+[
+  {
+    "elements": [
+      {
+        "name": "Generics",
+        "title": "Generics",
+        "type": "package",
+        "elements": [
+          {
+            "name": "TestOnlyGenerics",
+            "title": "TestOnlyGenerics",
+            "isAbstract": false,
+            "members": [],
+            "extends_": [],
+            "implements_": [],
+            "generics": "G1",
+            "stereotype": ""
+          },
+          [
+            [],
+            "'",
```

Comment:
this is a comment, and should be parsed as such. But I would ignore this for that pull request. Will fix with: https://github.com/Enteee/plantuml-parser/issues/46

Reply:
fixed